internal/store/nats.go

```diff
@@ -211,6 +211,11 @@ func (n *natsStore) Create(ctx context.Context, serverID uuid.UUID, condition *r
 	return err
 }
 
+// AddServer creates a server record in FleetDB
```

Comment:
In line with my comment above, we're never going to use NATS to store servers so we should avoid implementing even this much in the NATS repo.